gitpractice/task2/src/app/userModel.ts

```diff
@@ -0,0 +1,15 @@
+export class User{
+    public id: string;
+    public firstName: string;
+    public lastName: string;
+    public age: Number;
+    isDeleted:boolean;
+
+    constructor(id,firstname,lastname,age,isDeleted){
+        this.id=id;
+        this.firstName=firstname;
+        this.lastName=lastname;
+        this.age=age;
+        this.isDeleted=isDeleted;
+    }
+}
```

Comment:
this is a user model
check you task they have given the interface to use it
not the class with a constructor